spec/features/documents/report_spec.rb

```diff
@@ -147,15 +135,29 @@
       expect(page).to have_selector "a.launch-button[href='#{url2}']"
       expect(page).to have_selector "a.launch-button[href='#{url3}']"
     end
+    describe 'multiple users with the same username' do
+      scenario 'report should show report for "bar" runKey' do
+        url1 = doc_url(server, {recordid: dup_student1_doc.id, documentServer: 'https://www.example.com/', runKey: 'bar'})
+        url2 = doc_url(server, {recordid: dup_student2_doc.id, documentServer: 'https://www.example.com/', runKey: 'foo'})
+        visit report_path(owner: author.username, recordname: template.title, server: server, reportUser: dup_student1.username, runKey: 'bar')
+        expect(page).to have_selector('.launch-button', count: 1)
+        expect(page).to have_selector "a.launch-button[href='#{url1}']"
+      end
+      scenario 'report should show report for "foo" runKey' do
+        url1 = doc_url(server, {recordid: dup_student1_doc.id, documentServer: 'https://www.example.com/', runKey: 'bar'})
+        url2 = doc_url(server, {recordid: dup_student2_doc.id, documentServer: 'https://www.example.com/', runKey: 'foo'})
+        visit report_path(owner: author.username, recordname: template.title, server: server, reportUser: dup_student2.username, runKey: 'foo')
+        expect(page).to have_selector('.launch-button', count: 1)
+        expect(page).to have_selector "a.launch-button[href='#{url2}']"
+      end
+    end
```

Comment:
The above two tests are a little bit confusing.  

They make me wonder how `:owner` and `:reportUser` report parameters interact.

As near as I can tell, the `:owner_id` is always ignored.  If its used, its quite stealthy.  There are lots of places where it looks like its being used.  We use the `:reportUser` param to generate active record queries against a documents `:owner_id` for example.